install/pre-install-payload/containerd/payload.sh

```diff
@@ -22,12 +22,14 @@ function setup_env_for_arch() {
 		"linux/amd64") 
 			image="docker.io/library/centos"
 			kernel_arch="x86_64"
+			golang_arch="amd64"
 			;;
 		"linux/s390x")
 			image="docker.io/library/clefos"
 			kernel_arch="s390x"
+			golang_arch="s390x"
 			;;
-		(*) echo "$1 is not supported" && exit 1
+		(*) echo "$1 is not supported" 2> /dev/stderr && exit 1
```

Comment:
hmm... redirecting fd `2` to `stderr`? I guess you either meant `> /dev/stderr` or ` >&2`.

Reply:
Covered